src/docquery/find_leaf_nodes.js

```diff
@@ -64,13 +23,52 @@ function computeViewport() {
   };
 }
 
+function findLeafNodes(node) {
+  var textNodes = [];
+  var walk = document.createTreeWalker(
+    document.body,
+    NodeFilter.SHOW_TEXT,
+    null,
+    false
+  );
+
+  const vp = computeViewport();
```

Comment:
I don't think you're using `vp` at all

Reply:
Whoops that's before I realized reduce doesn't require an initial argument. Thanks